### PR TITLE
bring back missing event time

### PR DESCRIPTION
**Pull request by AMashoshyna**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
               {% assign location = site.data.locations[event.location][page.lang] %}
               <address class="upcoming-info__location">
                 <label class="upcoming-info__show-map" for="{{event.sortDate}}map" title="{{commonStrings.showMap}}">
-                  {{location.name}}, {{location.address}}{% if event.time %}, {{event.time}}{% endif %}
+                  {{location.name}}, {{location.address}}
```
Comment (tyv) [inline _layouts/default.html:36]: but here u deleted time...

Comment (AMashoshyna) [reply _layouts/default.html:36]: right, but there's one more  in '_event' include: 
https://github.com/listochkin/kyivjs/blob/gh-pages/_includes/event.html#L5
should I change the PR and leave both?

Comment (tyv) [reply _layouts/default.html:36]: ah, ok

**Comment by tyv** [comment]
@listochkin @sudodoki merge please

**Pull request merged**